source/views/transportation/other-modes/list.js

```diff
@@ -57,23 +56,17 @@ export class OtherModesView extends React.PureComponent<Props, State> {
 	}
 
 	refresh = async (): any => {
-		let start = Date.now()
 		this.setState(() => ({refreshing: true}))
-
-		await this.fetchData()
-
-		// wait 0.5 seconds – if we let it go at normal speed, it feels broken.
-		let elapsed = Date.now() - start
-		if (elapsed < 500) {
-			await delay(500 - elapsed)
-		}
-
+		await this.fetchData(true)
 		this.setState(() => ({refreshing: false}))
 	}
 
-	fetchData = async () => {
+	fetchData = async (reload?: boolean) => {
 		let {data: modes}: {data: Array<OtherModeType>} = await fetch(
 			transportationUrl,
+			{
+				delay: reload ? 500 : 0,
+			},
```

Comment:
```suggestion
			{delay: reload ? 500 : 0},
```

instead of the three-line version, please? :)